linux-tools/prelink/prelink.sh

```diff
@@ -89,14 +89,22 @@ function tc_local_setup()
 {
 	tc_exec_or_break $REQUIRED || return
 
-	tc_exist_or_break /etc/prelink.conf /etc/sysconfig/prelink || return
+	#tc_exist_or_break /etc/prelink.conf /etc/sysconfig/prelink || return
+	tc_exist_or_break /etc/prelink.conf /usr/sbin/prelink || return
 
 	tc_get_os_arch
 	userlibdir=/usr/lib
 	libdir=/lib/
 	[ $TC_OS_ARCH = "x86_64" ] || [ $TC_OS_ARCH = "ppc64" ] || [ $TC_OS_ARCH = "s390x" ] \
         && userlibdir=/usr/lib64/ && \
 		libdir=/lib64/
+	# On Ubuntu usrlib and lib paths are diff, so modified the code based on that
+	grep -i "ubuntu" /etc/*-release >/dev/null 2>&1
```

Comment:
Below change is specific for x86_64 arch. but prelink is available for power as well. 
Please modify this change to work on all archs.